src/tools/rbd_mirror/Replayer.cc

```diff
@@ -113,6 +113,33 @@ class ReplayerAdminSocketHook : public AdminSocketHook {
   Commands commands;
 };
 
+class MirrorStatusWatchCtx : public librados::WatchCtx2 {
```

Comment:
What is this class used for?  I don't believe you need to be watching an object in order to send it notifications.